public/preload.js

```diff
@@ -16,6 +16,26 @@ Sentry.init({
 })
 
 process.once('loaded', () => {
+  const oldAccessDataRaw = localStorage.getItem('wallet:accessData');
+  if (oldAccessDataRaw) {
+    // When migrating old wallets initialized with hardware wallets we need to clean the localStorage
+    const accessData = JSON.parse(oldAccessDataRaw);
+    const isHardware = localStorage.getItem('wallet:type') === 'hardware';
+    if (accessData.from_xpub || isHardware) {
+      const uniqueId = localStorage.getItem('app:uniqueId');
+      const backup = localStorage.getItem('wallet:backup');
```

Comment:
Why do you need this attribute? This marks if the software wallet backup has been done, isn't it?

Reply:
The `uniqueId` is good to have but the backup I don't see the need. I think we can keep some, the `started` attribute moves the startup screen to be the wallet type screen, for example.